### PR TITLE
TACOCAT cladding Properties

### PR DESCRIPTION
**Pull request by thurstonsd**

### CODE REVIEW

**Comment by lcarasik** [comment]
This pull request is intended to partially issue #62 by creating the dictionary for cladding properties.

File: LoadedTACO/src/Clad_Props.py
Comment (lcarasik) [inline LoadedTACO/src/Clad_Props.py:22]: Remove one of the extra lines

File: LoadedTACO/src/Clad_Props.py
Comment (lcarasik) [inline LoadedTACO/src/Clad_Props.py:49]: Fix the indention here for lines 49-51

File: LoadedTACO/src/Clad_Props.py
Comment (lcarasik) [inline LoadedTACO/src/Clad_Props.py:65]: Fix indentation for lines 61-64

**Review by lcarasik — CHANGES_REQUESTED** [review]
This has been reviewed, there are a few stylistic changes that need to be made before approval.

**Comment by thurstonsd** [comment]
I need to create a new branch on the computer and then send information to github again

**Comment by thurstonsd** [comment]
I need to create a new branch on the computer and then send information to github again

**Pull request closed without merging**
